jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -1049,6 +1070,12 @@ bool java_bytecode_languaget::convert_single_method(
   symbol_table_baset &symbol_table,
   optionalt<ci_lazy_methods_neededt> needed_lazy_methods)
 {
+  // Method is not in context, hence we don't convert it
```

Comment:
Slightly misleading comment. I would rephrase: `Do not convert if method is not in context`.